src/TesApi.Web/BatchScheduler.cs

```diff
@@ -1154,15 +1154,16 @@ private async Task<CloudTask> ConvertTesTaskToBatchTaskAsync(string taskId, TesT
 
             return cloudTask;
 
+            static string BashWrapShellArgument(string argument)
+                => $"'{argument.Replace(@"'", @"'\''")}'";
```

Comment:
Should this escape any special characters?

Reply:
This actually covers everything needed for the shell escaping. It does prevent using the command-line to obtain environmental variable values, though. But the goal is to get as close to calling the docker api as possible with the passed in values.